Add testing section to E-commerce and change template (#77)
GoogleCloudPlatform/click-to-deploy-solutions: ecommerce-webhosting/README.md

gcloud builds submit . --config cloudbuild.yaml
```

## Testing 

After you deployed the solution, you can check the resources created and see how they work together.

First, go to  [Cloud LoadBalancing](https://console.cloud.google.com/net-services/loadbalancing/list/loadBalancers) and click on the created Load Balancer.

![loadbalancer](assets/loadbalancer.png)

> You can see all the details of your Load Balancer, we will use our ip in Frontend section to access our teamplate.

Reviewer: ```suggestion
You can see all the details of the Load Balancer, and you can copy the Frontend IP section to access the application.
```

![loadbalancer](assets/loadbalancer.png)

> You can see all the details of the Load Balancer, and you can copy the Frontend IP section to access the application.

Reviewer: ```suggestion
You can see all the details of the Load Balancer, and you can copy the Frontend IP to access the application.
```

Author: fixed
